src/app/public/modules/radio/radio-group.component.ts

```diff
@@ -154,6 +172,15 @@ export class SkyRadioGroupComponent implements AfterContentInit, AfterViewInit,
     this.value = value;
   }
 
+  /**
+   * @internal
+   * Sets the disabled state of the control. Implemented as a part of ControlValueAccessor.
+   * @param isDisabled Whether the control should be disabled.
```

Comment:
Not sure if this line is going to show up anywhere, but if so, I'd change this slightly to follow the pattern in the docs: `Indicates whether to disable the control.`

Reply:
Done.